ibis/pyspark/compiler.py

```diff
@@ -1006,7 +977,9 @@ def compile_window_op(t, expr, scope, **kwargs):
         )
         pyspark_window = pyspark_window.rowsBetween(start, end)
 
-    result = t.translate(operand, scope, window=pyspark_window)
+    result = t.translate(
```

Comment:
@icexelloss could you provide a docstring for this function using numpy format? https://numpydoc.readthedocs.io/en/latest/format.html